.circleci/config.yml

```diff
@@ -15970,14 +15970,48 @@ commands:
         type: executor
         default: linux-e2e-executor
     steps:
-      - run:
-          name: Run E2e Tests
-          command: |
-            source .circleci/local_publish_helpers.sh
-            source $BASH_ENV
-            amplify version
-            retry runE2eTest
-          no_output_timeout: 90m
+      - when: null
+        condition:
+          equal:
+            - machine:
```

Comment:
maybe this will work in CCI, but this seems like a lot of stuff to be comparing to "parameters.os" and expecting equality

Reply:
nvm, I see this same thing is used in other places. LGTM